client/verta/tests/env_fixtures.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from verta._internal_utils.credentials import EmailCredentials, JWTCredentials
+from verta.credentials import EmailCredentials, JWTCredentials
```

Comment:
It looks like there are three different styles being used for this import in different files. We might want to pick one if `black` doesn't do it for us!
```python
from verta.credentials import EmailCredentials, JWTCredentials
```
```python
from verta.credentials import (EmailCredentials, JWTCredentials)
```
```python
from verta.credentials import (
    EmailCredentials,
    JWTCredentials,
)
```